app/views/foreman_ansible/job_templates/ansible_windows_updates.erb

```diff
@@ -0,0 +1,187 @@
+<%#
+name: Ansible Windows - Updates
```

Comment:
```suggestion
name: Install Windows Updates - Ansible Default
```

Reply:
You can also search and download updates, so Install would not really describe what you can do